website/site/content/platforms/arduinouno.md

```diff
@@ -0,0 +1,210 @@
+---
+title: "Arduino Uno"
+description: "See what you can do with the Arduino Uno: all the sensors and connectivity, examples, how it works and how the platform can be used."
+weight: 40
+draft: false
+toc: true
+menu:
+  main:
+    parent: Platforms
+    identifier: arduinouno
+    weight: 0
+---
+
+The Arduino Uno is a popular platform and is often used for the very beginning of getting in touch with embedded systems. 
```

Comment:
I think the introduction goes a bit too far. How about:

The [Arduino Uno](https://www.arduino.cc/en/Main/ArduinoBoardUno?setlang=en) is a popular platform for "physical computing" and first steps in embedded computing. It is based on an 8-bit Atmega microcontroller and ships with a powerful hardware abstraction library. 
Currently, the Mita Arduino Uno platform supports basic functionality, namely working with buttons and GPIO pins.